src/sqlancer/feldera/FelderaSchema.java

```diff
@@ -101,12 +64,9 @@ public static FelderaDataType getRandomNumericType() {
 
         public boolean isNumeric() {
             switch (this) {
-            case REAL:
-            case DOUBLE:
-            case TINYINT:
-            case SMALLINT:
+            case FLOAT:
```

Comment:
why this change?

Reply:
I changed this type to only represent the primitive types, so FLOAT represents DOUBLE, BOOL, and INT represents all the different sizes of the integer types.
This is similar to the implementation done by other databases in sqlancer.